includes/providers/gam/class-gam-model.php

```diff
@@ -740,8 +740,11 @@ public static function get_responsive_size_map( $sizes, $width_diff_ratio = 0.3,
 			foreach ( $sizes as $size ) {
 				$is_in_viewport     = $size[0] <= $viewport_width;
 				$is_above_threshold = false !== $width_threshold && $width_threshold <= $size[0];
-				$diff               = min( $viewport_width, $size[0] ) / max( $viewport_width, $size[0] );
-				$is_within_ratio    = ( 1 - $width_diff_ratio ) <= $diff;
+				if ( 0 == $size[0] ) {
```

Comment:
```suggestion
				if ( 0 === $size[0] ) {
```

Is there a reason to use `==` here?

Reply:
None, it's an oversight.